docs/functions.json

```diff
@@ -141,9 +141,51 @@
         ]
     },
     {
-        "signature": "datetime(text)",
-        "summary": "Parses `text` into a date using to the default date format.",
-        "detail": "An error is returned if the value can't be converted.",
+        "signature": "date(value)",
+        "summary": "Tries to convert `value` to a date.",
+        "detail": "If it is text then it will be parsed into a date using to the default date format.\nAn error is returned if the value can't be converted.",
```

Comment:
typo.. `using the`

Reply:
does this work for MM-DD-YYYY? and DD-MM-YYYY? depending on org format? might be good to have an example there.

Reply:
problem is all these examples run against the same default environment which happens to be YYY-MM-DD - could change that to DD-MM-YYYY